### ISSUE TITLE
OpenSCAD launch and rebuild error

### ISSUE THREAD

**Issue by podsvirov** [msys2/MINGW-packages#5097]
I update my environment:

> pacman -Syyuu

And install OpenSCAD:

> pacman -S mingw-w64-x86_64-openscad

I have error when try to launch openscad.exe over `mingw64`:
![msys2-mingw64-openscad-launch-error](https://user-images.githubusercontent.com/8400779/54716339-8d3f6400-4b66-11e9-99c6-3f8006f8b79e.png)
I try rebuild `mingw-w64-openscad` package but get a error in `uic` headers like:
```
In file included from src/Preferences.h:7,
                 from src/scintillaeditor.cpp:8:
objects/ui_Preferences.h:1168:117: error: no matching function for call to '_(const char [66], std::nullptr_t)'
         localizationCheckBox->setText(_("Enable user interface localization (requires restart of OpenSCAD)", nullptr));
```
UPDATES:
Looks like old source code incompatible with current environment, but with some fixes #5102 we can build OpenSCAD from latest source code from `master` branch via `mingw-w64-openscad-git` package.

**Comment by umarcor** [comment]
I guess this can be closed in favour of #7231...

**Comment by podsvirov** [comment]
Closed via #7906.
